purchase_fop_shipping/i18n/purchase_fop_shipping.pot

```diff
@@ -0,0 +1,93 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* purchase_fop_shipping
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 14.0\n"
```

Comment:
to be ask if need to change version by 15

Reply:
AFAIR this should be automatically updated by the ocabot